lib/events.js

```diff
@@ -186,24 +187,47 @@ function promisifyInputs(body, geoinfo, supporter) {
              )
     );
 
+    var processed_timelines = 0
+    var processed_impressions = 0
+    var processed_html = 0
+    
     /* this big debug noise is handy on the server */
     if(processed.timelines && processed.timelines[0] && processed.timelines[0].nonfeed)
         debug(" * non-newsfeed navigation: no content received");
-    else
+    else{
+	processed_timelines = _.size(processed.timelines)
+	processed_impressions =  _.size(processed.impressions)
+	processed_html = _.size(processed.htmls)
+    	
         debug(" * %d timelines %d impressions %d html %s",
-            _.size(processed.timelines),
-            _.size(processed.impressions), _.size(processed.htmls),
+	      processed_timelines,
+	      processed_impressions,
+	      processed_html,
             _.get(processed.timelines[0], 'tagId') ?
                 "tagId " + processed.timelines[0].tagId :
                 ""
             );
-
+    }
+    
     debug(" * user %d [%s] last activity %s (%s ago) %s",
         supporter.userId, geoinfo,
         moment(supporter.lastActivity).format("HH:mm DD/MM"),
         moment.duration(moment.utc()-moment(supporter.lastActivity)).humanize(),
-        supporter.version);
-
+          supporter.version);
+    
+    var ts = Math.round((new Date()).getTime() / 1000);
+
+    echoes.echo({
+	id: ts,
+	index: "fbtrex_users",
+	support_id: support.userId,
+	geo: geoinfo,
+	last_activity: support.lastActivity,
+	timelines: processed_timelines,
+	impressions: processed_impressions,
+	htmls: processed_html
+    })
+    
```

Comment:
@joxer thanks, well done, I understand the point. I'll use the echoes in other components of the system.
is `index` the identifier of this log message?
and `id` has a risk of (very unlikely) duplication. If this happen, what would happen?
I'll merge this commit but I'll open an issue before consider this experiment complete, would be mandatory do a pseudonym of userId.